src/screens/ImportWalletScreen.tsx

```diff
@@ -154,15 +171,30 @@ export class ImportWalletScreen extends PureComponent<Props, State> {
       });
     } else {
       newWallet.setLabel(this.state.label || i18n.wallets.import.imported + ' ' + newWallet.typeReadable);
+      checkSubscription([newWallet], email);
       importWallet(newWallet, {
         onSuccess: () => {
-          // isNotificationEmailSet
-          //   ? this.props.navigation.navigate(Route.ReceiveNotificationsConfirmation, {
-          //       address: email,
-          //       flowType: ConfirmAddressFlowType.RECEIVE_NOTIFICATIONS_CONFIRMATION_IMPORT,
-          //     })
-          //   : // TODO in import wallet logic
-          this.showSuccessImportMessageScreen();
+          this.props.navigation.navigate(Route.CreateWalletSuccess, {
```

Comment:
check the flow in wimsical i dont see CreateWalletSuccess there

Reply:
Still, I doubt that we don't want to show secrets to the user